neo/Wallets/Wallet.cs

```diff
@@ -263,8 +263,23 @@ public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null
                     script = sb.ToArray();
                 }
                 if (balances_gas is null)
-                    balances_gas = accounts.Select(p => (Account: p, Value: NativeContract.GAS.BalanceOf(snapshot, p))).Where(p => p.Value.Sign > 0).ToList();
-                TransactionAttribute[] attributes = cosigners.Select(p => new TransactionAttribute { Usage = TransactionAttributeUsage.Cosigner, Data = p.ToArray() }).ToArray();
+                    balances_gas = accounts.Select(p => (Account: p, Value: NativeContract.GAS.BalanceOf(snapshot, p))).Where(p => p.Value.Sign > 0).ToList();
+
+                var attributes = cosigners.Select(p =>
+                        new TransactionAttribute
+                        {
+                            Usage = TransactionAttributeUsage.Cosigner,
+                            Data = new CosignerUsage
+                            {
+                                Scope = new WitnessScope
+                                {
+                                    // default access for transfers should be 'root-access'
+                                    Type = WitnessScopeType.RootAccess,
```

Comment:
@igormcoelho is not possible to be a RootAccess, because you build a scriptA -> and call the native contract (ScriptB)

Reply:
Its incorrect the definition of RootAccess, I just saw... it must be CallingScriptHash == Entry, not Current == Entry